tests/unit_tests/test_wave_03.py

```diff
@@ -131,7 +131,11 @@ def test_swap_items_from_their_empty_returns_false():
 
     result = fatimah.swap_items(jolie, item_b, nobodys_item)
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
+    assert result is False
+    assert len(fatimah.inventory) == 3
+    assert len(jolie.inventory) == 0
```

Comment:
Great assertions, I would also suggest checking `fatima`'s inventory contents, since checks like the length being correct do not guarantee that the individual elements are still what we expect.